.gitlab-ci.yml

```diff
@@ -622,6 +622,40 @@ build/icpx/igpu/release/static:
     ONEAPI_DEVICE_SELECTOR: "*:gpu"
     BUILD_HWLOC: "OFF"
 
+# windows jobs: Release shared
```

Comment:
I think it is the powershell?
maybe some notes will help because powershell is a bit different from the cmd

Reply:
good point, yes it is powershell